index.html

```diff
@@ -0,0 +1,390 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta http-equiv="content-type" content="text/html" ; charset="utf8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="container">
+        <div class="header__top">
+                <p class="header__top__logo">Your Logo</p>            
+                <div class="header__top__menu">
```

Comment:
You need to use `<nav>` tag instead of `<div class="header__top__menu">`

Reply:
done